docs/_sass/_search-bar.scss

```diff
@@ -252,7 +252,7 @@ label.search-label {
 
 /* Change the color & background of Google Search of Other Page */
 .gsc-resultsbox-visible .gsc-results .gsc-cursor-box .gsc-cursor-page {
-    background-color: $color-highlightBG;
+    background-color: $color-borders;
```

Comment:
Rather than use borders, can we create a var in colors.scss for our button background?

Same with the hover color, that should be a var we create. The value would be #2C6755